### DISCUSSION TITLE
Our Brand

### DISCUSSION THREAD

**Discussion by confused-Techie** [pulsar-edit/.github#66] [Documentation & Website]
Alright everyone, since we are currently, seemingly, on track to receive our new icons from the designer soon I feel we need to talk about a few other aspects of our branding.

Having the Icon and animations is one thing, which I'm excited that we will be having soon, but beyond that we still need to figure out one key thing, colour.

## Colour 

We need to determine the colour we want for our brand. I don't just mean the logo itself or it's dev/stable/nightly version and whatnot. I mean the colours for our whole brand. The background colour of our various websites, of buttons on our readme, highlight colour on Open Collective and so forth.

Really from the research I've done and talking to others that are more knowledgeable about design what we should do is try to find 4 different colours to choose as our brand. Two primary colours and two secondary. Now ideally once we have a few options we can open up a poll over on Discord to choose which one we like the best.

## Other Brand Todos 

Otherwise some things we should at least start discussing, is banners. Since ideally we could have along with our logo a banner to display where relevant. And we should open discussion about the background or other attributes of any banner. Such as do we want it overlayed with stars? Abstract shapes? Or just a solid colour?

Just wanted to get a discussion going here, so that ideally, once we get our icon in we already can start rebranding everything important before mid December.

**Comment by preland** [comment]
I don't have that much experience with brand design, but this is what I am thinking:

Use #F0BC61 (a sun-like light orange) as primary, with #3F1558 (a dark purple, inspired by the current logo for Pulsar) as a secondary/contrasting color (the two could be role-reversed if need be). This color scheme would likely fit best for branding, front/landing page, as well as community announcements.

Here is a link that shows this look off (you can also change the colors and test different setups in realtime):  https://color.review/check/F0BC61-3F1558

For a more neutral color scheme, the colors #B8C1F9 (a muted lavender) as primary, with #3A2B43 (a more modest version of the dark purple from before) could be used. This color scheme would probably be a good fit for documentation, and potentially (big emphasis on potentially here) could be used as a default color scheme for the app itself, since this color scheme is very readable, non-invasive, and would give the app a unique look to it.

Here is a link that shows the above look off (you can also change the colors and test different setups in realtime):  https://color.review/check/B8C1F9-3A2B43

Honorable mention: While I think the 2nd isn't glaring bright, it could still be made even darker while still maintaining the overall look and contrast of the 2nd. Here is a link to this color scheme if you are interested: https://color.review/check/B8C1F9-3A2B43

Any feedback on these suggestions would be helpful.

**Reply by confused-Techie** [reply]
For reference to make my points easier:
1) [#F0BC61 - #3F1558](https://color.review/check/F0BC61-3F1558)
2) [#B8C1F9 - #3A2B43](https://color.review/check/B8C1F9-3A2B43)

Also I think you're honorable mention is the same link as the above section

---

Personally I love 2, and would prefer that used in most instances, the colour is light, and inviting plus from my little knowledge in design, both blue and purple are intended to be relaxing colours, so perfect for a page of documentation or the editor itself. My only concern would be that while their contrast is high enough, if it would become a strain on the eyes after prolong usage? Not experiencing it myself, but just something to consider.

As for 1 for the logo, I love the chosen purple and would vote myself to have that as the primary colour. And for the goldish, while it does work there would be interested to see if there's any other options anyone has, as I wouldn't say it was striking if that makes sense.

But thanks for responding, these look like fantastic suggestions!

**Comment by Daeraxa** [comment]
Had a quick play with a colour palette thing and came up with this.
The first two are just shades of purple that are similar yet stand out from each other for gradients etc.
The blue is kind of meant to be like the jet as featured on my famous icon as well as just a strong contrast.
The last two colours are just taken from Pulsar's default light and dark theme `@base-background-color`.
![image](https://user-images.githubusercontent.com/58074586/201992676-b78a05cb-dc3d-47f8-a76b-7f016c75027b.png)



The two grey shades can be swapped around for "dark" vs "light" theming.

e.g. 

https://color.review/check/E8EAED-282C34  vs  https://color.review/check/282C34-E8EAED 

A very rough example of the "dark" version of it on the website (I don't think this is a good look in itself, it would need proper work but just wanted to shove the colours into the same image to get an idea of the palette):
![image](https://user-images.githubusercontent.com/58074586/201990448-364b6a4c-c364-42ed-b522-0fae9d752058.png)

## "Light theme" docs:
![image](https://user-images.githubusercontent.com/58074586/201993240-9a6d1e8a-54a3-4cb4-90a9-fec5ee3803ab.png)

## "Dark theme" docs:
![image](https://user-images.githubusercontent.com/58074586/201993641-d4b69a9c-4b75-4718-9a41-9d0cb67a2666.png)

**Reply by confused-Techie** [reply]
This does look nice, and is good to see it laid out properly. Although the only change I'd personally want to see is replacing the purple used here with @preland's suggestion. although not sure how well it'll go with everything else here

**Comment by confused-Techie** [comment]
For anyone looking at this Discussion, we are now voting on what colours to go with for our brand, please go ahead and [vote](https://discord.com/channels/992103415163396136/1042598024007925771) for what you like best
